### ISSUE TITLE
Variable number of items

### ISSUE THREAD

**Issue by jerryjohnjacob** [sasikumargn/sales_tax#3]
Need to be able to add more or fewer items. Right now, the count is always locked to 5. It works for fewer items, yes, but it's not an elegant solution.

**Comment by sasikumargn** [comment]
@jerryjohnjacob : Yes, we can't add more or fewer items now. The assessment UI page had 5 Items to display it. so, I created with default 5 items. We can create add more feature as well.

**Comment by jerryjohnjacob** [comment]
@sasikumargn Yes please. Can you make that change?
